apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue

```diff
@@ -59,7 +82,9 @@ const props = defineProps<{
 const emit = defineEmits<{(e: 'update:isValid', isValid: boolean): void;}>();
 
 const state = reactive({
+    isVisibleTitle: computed<boolean>(() => (!props.hasMetadata && !Object.keys(state.schema ?? {}).length)),
```

Comment:
suggestion: What about to add `showTitleIfNoSchema` prop, and replace `!props.hasMetadata` with `props.showTitleIfNoSchema`?